src/dotnet-core-uninstall/Windows/RegistryQuery.cs

```diff
@@ -62,11 +68,10 @@ private static bool IsNetCoreBundle(RegistryKey uninstallKey)
 
             return IsNetCoreBundle(uninstallKey.GetValue("DisplayName") as string,
                 uninstallKey.GetValue("DisplayVersion") as string,
-                uninstallKey.GetValue("UninstallString") as string,
-                uninstallKey.GetValue("BundleVersion") as string);
```

Comment:
Any reason for removing the BundleVersion. All the BundleXXX properties are present for bundles, but absent for MSIs, which means you could potentially pick up MSIs and classify them as bundles.

Reply:
I think this was leftover from debugging, I'll add it back